btc-lsp/src/BtcLsp/Storage/Model/Block.hs

```diff
@@ -69,10 +71,42 @@ getLatestSql =
         Psql.LimitTo 1
       ]
 
-getBlock :: (MonadIO m) => BlkHeight -> ReaderT Psql.SqlBackend m [Entity Block]
-getBlock blkHeight = do
+getBlockByHeightSql :: (MonadIO m) => BlkHeight -> ReaderT Psql.SqlBackend m [Entity Block]
+getBlockByHeightSql blkHeight = do
   Psql.select $
     Psql.from $ \row -> do
-      Psql.where_
+      Psql.where_ $
         (row Psql.^. BlockHeight Psql.==. Psql.val blkHeight)
+          Psql.&&. ( row Psql.^. BlockStatus
+                       Psql.==. Psql.val BlkConfirmed
+                   )
       pure row
+
+
+getOrphanBlocksHigherSql :: (MonadIO m) => BlkHeight -> ReaderT Psql.SqlBackend m [Entity Block]
+getOrphanBlocksHigherSql blkHeight = do
+  Psql.select $
+    Psql.from $ \row -> do
+      Psql.where_ $
+        (row Psql.^. BlockHeight Psql.>. Psql.val blkHeight)
+          Psql.&&. ( row Psql.^. BlockStatus
+                       Psql.==. Psql.val BlkConfirmed
```

Comment:
EIther function name or block status are incorrect.  getOrphan but status is confirmed. Is that correct?